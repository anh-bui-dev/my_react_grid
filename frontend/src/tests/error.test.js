import { shallow, mount } from 'enzyme';
import React, { Component } from 'react';
import Error from '../components/error';

describe('Error', () => {
  it('Should render correctly', () => {
    const wrapper = shallow(<Error />);
    expect(wrapper).toMatchSnapshot();
  });
});