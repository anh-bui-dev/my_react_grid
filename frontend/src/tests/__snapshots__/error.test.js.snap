// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Should render correctly 1`] = `""`;
